### DISCUSSION TITLE
2024-06-10(월)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#124] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- my order 페이지 UI

### 해야할 일
- my info 페이지 UI
- my review write 페이지 UI
- 인증 처리 API 연결 후 확인

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- Toast 컴포넌트 레이아웃에 따라 뜨는 위치 조정 작업 (모달 수정 필요)

### 해야할 일
- 무한스크롤 테스트
- 홈, 검색 페이지 API 작업

### 어려운 일
- 파이팅!💪

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
bottomSheet
페이지 작업

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 결제성공페이지
### 해야할 일
- 결제실패페이지
### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일
별점 컴포넌트 text 추가

### 해야할 일
마이페이지 -> 리뷰 상세 페이지

### 어려운 일
